src/event-listeners/index.js

```diff
@@ -40,7 +40,11 @@ import {
 import layoutFormula from '../drawing/layoutFormula'
 
 export default function setupEventListeners(store, canvas) {
-
+  let convertedClick 
+  let goalAddresses=null
+  let boolean=false;
+  let x=0;
+  let y = 0;
```

Comment:
let's avoid introducing these variables here. They should be added as new properties onto the `src/mouse/reducer.js` slice of the redux state, and modified using redux actions. @alex1812r has experience relating to this I think right, or @JhonatanHern